src/main/java/com/jcabi/github/mock/MkPull.java

```
@@ -130,6 +131,13 @@ public void merge(
     }
 
     @Override
+    public MergeState merge(
+        @NotNull(message = "message can't be NULL") final String msg,
+        final String sha) throws IOException {
+        return null;
```

@sonOfRa it's better to throw UnsupportedOperationException here